app-crypt/kbfs/kbfs-5.8.1.ebuild

```diff
@@ -1,13 +1,13 @@
-# Copyright 1999-2019 Gentoo Authors
+# Copyright 1999-2022 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=7
 
 inherit golang-build systemd
```

Comment:
Is there any way we can get this off `golang-build` and use `go-module` instead? This would let us bump it up to EAPI=8.

Reply:
Can we use EGO_SUM for that? If so, I can give it a try.

Reply:
I was looking through the golang & CVEs for this and the last-rites call, so far it looks like you're on track.

everything else looks good, including the fix for KBFS nags ~ *[and actually supporting KBFS once that's added]* ~ if you have time to look at EGO_SUM I wasn't able to research that, and it would be duplicated work on my end so if you know what needs done there and can get it to EAPI 8, very cool.

Thanks for having a look before this hit last-rites, because if not I was going to be busy with it myself 😅

Reply:
EGO_SUM is deprecated and we should ideally use a vendor tarball instead. I'll be happy to mirror it into my devspace.

Reply:
@ajakk I'll give it a try this weekend, will update this PR if I can get it to work.
(tbh the EGU_SUM deprecation is a pain in the rear and a mistake imho, but that's a different discussion, I'll try to find the relevant bug and join the discussion there).